be/src/exec/pipeline/exchange/sink_buffer.cpp

```diff
@@ -407,17 +402,7 @@ Status SinkBuffer::_try_to_send_rpc(const TUniqueId& instance_id, const std::fun
         closure->cntl.Reset();
         closure->cntl.set_timeout_ms(_brpc_timeout_ms);
 
-        Status st;
-        if (bthread_self()) {
-            st = _send_rpc(closure, request);
-        } else {
-            // When the driver worker thread sends request and creates the protobuf request,
-            // also use process_mem_tracker to record the memory of the protobuf request.
-            SCOPED_THREAD_LOCAL_MEM_TRACKER_SETTER(nullptr);
```

Comment:
Why is this branch needless anymore?

Reply:
All the updation of `tls_mem_tracker` within bthread will be simply ignored. So bthread only has process level mem_tracker, and no extra work required here.

Reply:
@liuyehcf  The protobuf request is allocated in `_send_rpc`, and released in bthreads (`DisposableClosure::run`).
`_send_rpc` can be invoked by driver threads (`ExchangeSinkOperator::push_chnk`) or bthreads (RPC response callback).
Therefore, when it is invoked by driver threads, it must use process mem tracker manually here, to make sure memory allocation and release use the same mem tracker.

Reply:
Oh, I got it, this should be remained.